### PR TITLE
Announce Keptn 0.16.0

### PR DESCRIPTION
**Pull request by thisthat**
Signed-off-by: Giovanni Liva <giovanni.liva@dynatrace.com>

### CODE REVIEW

File: content/docs/0.16.x/operate/upgrade/index.md [hunk 1]
```diff
@@ -9,61 +9,15 @@ aliases:
...
-
-   ```console
-   helm repo update
-   helm upgrade jmeter-service https://github.com/keptn/keptn/releases/download/0.16.0/jmeter-service-0.16.0.tgz -n keptn --create-namespace --wait --reuse-values
-   helm upgrade helm-service https://github.com/keptn/keptn/releases/download/0.16.0/helm-service-0.16.0.tgz -n keptn --create-namespace --wait --reuse-values
-   ```
-
-</p>
-</details>
+With Keptn 0.16.x, every Keptn project requires an upstream. Hence, before upgrading Keptn it is required to (i) attach an upstream to your Keptn projects and (ii) do a [backup](..//backup_and_restore/#back-up-configuration-service).
```
Comment (StackScribe) [inline content/docs/0.16.x/operate/upgrade/index.md:12]:
```suggestion
With Keptn 0.16.x, every Keptn project requires an [upstream Git repository](../manage/git_upstream). Hence, before upgrading Keptn. you must (i) attach an upstream to your Keptn projects and (ii) do a [backup](..//backup_and_restore/#back-up-configuration-service).
```

File: content/docs/0.16.x/operate/upgrade/index.md [hunk 2]
```diff
@@ -9,61 +9,15 @@ aliases:
...
+If you set an upstream for all your Keptn projects, no additional steps are required.
```
Comment (StackScribe) [inline content/docs/0.16.x/operate/upgrade/index.md:13]:
```suggestion
If you already have an upstream for all your Keptn projects, no additional steps are required.
```

File: content/docs/0.16.x/operate/upgrade/index.md [hunk 3]
```diff
@@ -9,61 +9,15 @@ aliases:
...
+
+Suppose you need the additional features provided by the *resource-service*,  such as HTTPS/SSH or Proxy, to configure your Keptn project with an upstream. In that case,
+you can also deploy the *resource-service* and configure the Git repositories later. For this, a backup is necessary.
+
+1. Back up of the [configuration-service](../backup_and_restore/#back-up-configuration-service).
```
Comment (StackScribe) [inline content/docs/0.16.x/operate/upgrade/index.md:18]:
```suggestion
1. Back up the [configuration-service](../backup_and_restore/#back-up-configuration-service).
```

File: content/docs/news/release_announcements/keptn-0160/index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+title: Keptn 0.16.0
+weight: 72
+---
+
+# Release Notes 0.16.0
+
+Keptn 0.16.0 provides the possibility to upgrade your Keptn installation without any downtime. For this, Keptn requires each project to have an upstream.
```
Comment (StackScribe) [inline content/docs/news/release_announcements/keptn-0160/index.md:8]:
```suggestion
Keptn 0.16.0 provides the possibility to upgrade your Keptn installation without any downtime. For this, Keptn requires each project to have an [upstream Git repository](../manage/git_upstream).
```

File: content/docs/news/release_announcements/keptn-0160/index.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+Make sure to check the [upgrade instructions](../../../0.16.x/operate/upgrade/) to do not lose any data.
```
Comment (StackScribe) [inline content/docs/news/release_announcements/keptn-0160/index.md:9]:
```suggestion
Make sure to check the [upgrade instructions](../../../0.16.x/operate/upgrade/) to ensure that you do not lose any data.
```

File: content/docs/news/release_announcements/keptn-0160/index.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+
+---
+
+**Key announcements**:
+
+:tada: *Zero downtime upgrades*: Operators can upgrade Keptn without downtime.
+
+:star: *Performance gains in backend and frontend*: Performance implications when working with resource files have been resolved due to a new backend service. Besides, the Keptn Bridge received performance improvements with an adapted polling behavior.
```
Comment (StackScribe) [inline content/docs/news/release_announcements/keptn-0160/index.md:18]:
```suggestion
:star: *Performance gains in backend and frontend*: Performance implications when working with resource files have been resolved due to a new backend service. In addition, the Keptn Bridge performance is improved bu an adapted polling behavior.
```

File: content/docs/news/release_announcements/keptn-0160/index.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+:sparkles: *(experimental) New heatmap for SLI breakdown*: Keptn Bridge is leveraging a new rendering library that offers more flexibility for displaying the SLI breakdown. If you want to try it out, you can enable it by setting the Helm value `control-plane.bridge.d3heatmap.enabled` to `true`.
```
Comment (StackScribe) [inline content/docs/news/release_announcements/keptn-0160/index.md:20]:
```suggestion
:sparkles: *(experimental) New heatmap for SLI breakdown*: Keptn Bridge is leveraging a new rendering library that offers more flexibility for displaying the SLI breakdown. To enable it, set the Helm value `control-plane.bridge.d3heatmap.enabled` to `true`.
```

**Comment by thisthat** [comment]
Thank you @StackScribe  for the feedback 🙌

**Review by StackScribe — APPROVED** [review]
A few suggestions for your consideration but it looks good.

**Pull request merged**